systemtest/src/test/java/io/strimzi/systemtest/upgrade/regular/StrimziUpgradeST.java

```diff
@@ -109,54 +109,19 @@ void testUpgradeAcrossVersionsWithUnsupportedKafkaVersion() throws IOException {
         final TestStorage testStorage = new TestStorage(ResourceManager.getTestContext());
         UpgradeKafkaVersion upgradeKafkaVersion = UpgradeKafkaVersion.getKafkaWithVersionFromUrl(acrossUpgradeData.getFromKafkaVersionsUrl(), acrossUpgradeData.getStartingKafkaVersion());
 
-        // Setup env
```

Comment:
Can we have some consistency in the comments?
* They seem to be in all other tests so it might make the most sense to keep them?
* I can see that they do not add much value in most cases (because surprise, the `setupEnv...` method sets up the "env"). But if we want to remove them as not useful, it would be best to use separate PR and do it everywhere. 

But this PR seems to deal with them in a strange way - it removes them here and in `testUpgradeAcrossVersionsWithNoKafkaVersion`. But keeps them in `performUpgradeAcrossVersions`. Maybe we can keep them everywhere for now and get bak to it later if needed?